crates/release_plz_core/src/repo_url.rs

```diff
@@ -32,10 +33,19 @@ impl RepoUrl {
         })
     }
 
+    pub fn from_repo(repo: &Repo) -> Result<Self, anyhow::Error> {
+        let url = repo.origin_url().context("cannot determine origin url")?;
+        RepoUrl::new(&url)
+    }
+
     pub fn is_on_github(&self) -> bool {
         self.host.contains("github")
     }
 
+    pub fn is_on_gitea(&self) -> bool {
```

Comment:
Gitea repositories can be self-hosted in domains different from "gitea".
We use [this](https://github.com/MarcoIeni/release-plz/blob/1b06fa3c369dfbd6edafdb92e9d1f20ac3953d4a/crates/release_plz/src/args/release_pr.rs#L32) method in the release-pr workflow to determine if the repo is in gitea or github.

So you need to add the `backend` command line flag.